agent/php_includes.h

```diff
@@ -53,6 +53,10 @@
 #define ZEND_8_0_X_API_NO 20200930
 #define ZEND_8_1_X_API_NO 20210902
 
+#if ZEND_MODULE_API_NO >= ZEND_8_0_X_API_NO /* PHP8+ */
```

Comment:
Maybe add `OVERWRITE_ZEND_EXECUTE_DATA` so we know this functionality is unique to OAPI?